Mathlib/Algebra/Category/CoalgebraCat/Basic.lean

```diff
@@ -177,3 +179,10 @@ def toCoalgEquiv (i : X ≅ Y) : X ≃ₗc[R] Y :=
   rfl
 
 end CategoryTheory.Iso
+
+instance CoalgebraCat.forget_reflects_isos :
+    (forget (CoalgebraCat.{u} R)).ReflectsIsomorphisms where
+  reflects {X Y} f _ := by
+    let i := asIso ((forget (CoalgebraCat.{u} R)).map f)
```

Comment:
also I guess this would be more general
```suggestion
    (forget (CoalgebraCat.{v} R)).ReflectsIsomorphisms where
  reflects {X Y} f _ := by
    let i := asIso ((forget (CoalgebraCat.{v} R)).map f)
```

Reply:
Thanks!